scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -26,26 +26,31 @@ class RemoveUnused(config: RemoveUnusedConfig)
   def this() = this(RemoveUnusedConfig.default)
 
   override def description: String =
-    "Removes unused imports and terms that reported by the compiler under -Ywarn-unused"
+    "Removes unused imports and terms that reported by the compiler under -Wunused"
   override def isRewrite: Boolean = true
 
   private def warnUnusedPrefix = List("-Wunused", "-Ywarn-unused")
   private def warnUnusedString = List("-Xlint", "-Xlint:unused")
   override def withConfiguration(config: Configuration): Configured[Rule] = {
+    def diagnosticsUnavailableInSemanticdb(version: String) =
+      version.startsWith("3.0") ||
+        version.startsWith("3.1") ||
+        version.startsWith("3.2") ||
+        version.startsWith("3.3")
```

Comment:
[3.3.3 is likely to get the semanticdb fix](https://github.com/lampepfl/dotty/pull/17835#issuecomment-1862460194) but I'll update when I see it happening in a RC